### PR TITLE
[Fli iam#1474] Delete from tree nodes

### PR DESCRIPTION
**Pull request by youennmerel**
From tree displayed in study and subject details, the following elements can be deleted via the node dropdown menu :
- acquisition
  - examination
    - dataset
      - processing
        - processed dataset
- equipment
- study card

The "delete" option is displayed only if the authenticated user is allowed to do so (see https://github.com/fli-iam/shanoir-ng/wiki/Roles-and-rights-specification#synthesis-table)

Note that two bug related to equipment and study card deletion are currently open :
- https://github.com/fli-iam/shanoir-ng/issues/1688
- https://github.com/fli-iam/shanoir-ng/issues/1689

### CODE REVIEW

File: shanoir-ng-front/src/app/examinations/tree/examination-node.component.ts [hunk 1]
```diff
@@ -161,25 +165,44 @@ export class ExaminationNodeComponent implements OnChanges {
...
+            processing.outputDatasets ? processing.outputDatasets.map(ds => this.mapDatasetNode(ds, true)) : [],
+            this.node.canDelete
         );
     }
+
+    deleteExamination() {
+        this.examinationService.get(this.node.id).then(entity => {
+            this.examinationService.deleteWithConfirmDialog(this.node.title, entity).then(deleted => {
+                if (deleted) {
+                    console.log("DELETED !")
```
Comment (jcomedouteau) [inline shanoir-ng-front/src/app/examinations/tree/examination-node.component.ts:197]: Remove logs

**Comment by jcomedouteau** [comment]
Makes me sad to duplicate the code as such, but it's working fine for me !
Just remove the logs and we are good !

File: shanoir-ng-front/src/app/examinations/tree/examination-node.component.ts (hunk 1 above)
Comment (youennmerel) [reply shanoir-ng-front/src/app/examinations/tree/examination-node.component.ts]: :hand_over_mouth:

**Comment by youennmerel** [comment]
> Makes me sad to duplicate the code as such, but it's working fine for me ! Just remove the logs and we are good !

Yeah, it would be great to have more generic components and inherited methods for tree nodes...
Maybe I could have added a NodeComponent class with common methods inherited by all nodes ? 

That being said, I've removed the log.

**Pull request merged**